components/sendoso/sources/new-send/new-send.mjs

```diff
@@ -0,0 +1,58 @@
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  type: "source",
+  name: "New Send",
+  key: "sendoso-new-send",
+  description: "Emit new event when a new send is created. [See docs here](https://sendoso.docs.apiary.io/#reference/send-management/get-all-sent-giftsitems/get-sent-gifts)",
```

Comment:
@GTFalcao I know it may be strange, but Sendoso handles Sends, so this action creates a Send, it's not a generic send, but the send object. However, I don't see any problems with changing to your suggestion "New Send Created".